nft_ingester/src/program_transformers/bubblegum/burn.rs

```diff
@@ -26,27 +33,32 @@ where
             &mpl_bubblegum::ID,
         );
         debug!("Indexing burn for asset id: {:?}", asset_id);
-        let id_bytes = asset_id.to_bytes().to_vec();
-        let asset_to_update = asset::ActiveModel {
-            id: Unchanged(id_bytes.clone()),
+        let id_bytes = asset_id.to_bytes();
+
+        let asset_model = asset::ActiveModel {
+            id: Set(id_bytes.to_vec()),
             burnt: Set(true),
-            seq: Set(seq as i64),
```

Comment:
If a burn transaction comes in out of order, it needs to set the `burnt` flag (because nobody else will as it's not a leaf schema thing), but we don't want it to overwrite `seq` with a potentially older value.  Thus `seq` is now updated later by `upsert_asset_with_seq`, which will only update `seq` if `excluded.seq > asset.seq`.

Reply:
Hmm gotcha.

Can there be any events for a cNFT after its burnt?

Reply:
No there cannot be any events for a cNFT after its burnt.

1. It is intended that burn destroys the NFT, which it does by replacing that leaf with zeros, leaving an empty hole in the tree.
2. If someone tried to do something with the leaf after burning it, any of those operations fail because the "previous leaf" is determined in other instructions by id, owner, data_hash, etc. which would not match the leaf of all zeros.